src/lv_init.c

```diff
@@ -285,6 +286,10 @@ void lv_init(void)
     lv_fs_arduino_sd_init();
 #endif
 
+#if LV_USE_JPEGDEC
+    lv_jpegdec_init();
```

Comment:
Please add also `lv_jpegdec_deinit()` in the `deinit()` function

Reply:
I can add that @PGNetHun , but none of the other imaging codecs call deinit(). Why is that?